core_perception/ekf_localizer/launch/ekf_localizer.launch

```diff
@@ -3,10 +3,8 @@
   <arg name="show_debug_info" default="false"/>
   <arg name="enable_yaw_bias_estimation" default="True"/>
   <arg name="predict_frequency" default="50.0"/>
-  <arg name="tf_rate" default="10.0"/>
```

Comment:
Remove this argument from the call in carma-platform/carma/launch/localization.launch file

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/pull/1335